README.md

```diff
@@ -1,130 +1,144 @@
 
-The demo applications are tested on the [Iguazio's Data Science PaaS](https://www.iguazio.com/), and use Iguazio's shared data fabric (v3io), and can be modified to work with any shared file storage by replacing the `apply(v3io_mount())` calls with other Kubeflow volume modifiers.
-You can request a [free trial of Iguazio PaaS](https://www.iguazio.com/lp/14-day-free-trial-in-the-cloud/).
+The demo applications are tested on the [Iguazio Data Science Platform](https://www.iguazio.com/) ("the platform") and use its shared data fabric, which is accessible via the `v3io` file-system mount; if you're not already a platform user, [request a free trial](https://www.iguazio.com/lp/14-day-free-trial-in-the-cloud/).
+You can also modify the code to work with any shared file storage by replacing the `apply(v3io_mount())` calls with any other Kubeflow volume modifier.
 
-### Examples
+<a id="general-ml-workflow"></a>
+### General ML Workflow
 
-* [scikit-learn Pipeline with AutoML](#end-to-end-data-prep--scikit-learn-pipeline-with-automl-iris-data-set)
-* [Image Classification Using Distributed Training (Horovod)](#image-classification-using-distributed-training-horovod)
-* [Real-Time Face Recognition with Re-enforced Learning](#real-time-face-recognition-with-re-enforced-learning)
-* [Predictive Network/Telemetry Monitoring](#predictive-networktelemetry-monitoring)
-* [Real-time Customer Churn Prediction](#real-time-customer-churn-prediction-kaggle-telco-churn-dataset)
-
-<a id="prerequisites"></a>
-### Prerequisites
+The provided demos implement some or all of the ML workflow steps illustrated in the following image:
 
-- A Kubernetes cluster with preinstalled operators/CRDs for Horovod, Nuclio, Spark (depending on the specific demo).
-- MLRun Service installed in the cluster, [see instructions](https://github.com/mlrun/mlrun/blob/master/README.md#installation).
-- Shared file or object storage for the data/artifacts.
+<br><p align="center"><img src="./docs/mlrun-pipeline.png" width="800"/></p><br>
 
-<a id="general-ml-flow"></a>
-### The General ML Pipeline Flow
+<a id="prerequisites"></a>
+## Prerequisites
 
-The various demos follow some or all of the steps shown in the diagram below:
+To run the MLRun demos, first do the following:
 
-<br><p align="center"><img src="./docs/mlrun-pipeline.png" width="800"/></p><br>
+- Prepare a Kubernetes cluster with preinstalled operators or custom resources (CRDs) for Horovod and/or Nuclio, depending on the demos that you wish to run.
+- Install an MLRun service on your cluster.
+  See the instructions in the [MLRun documentation](https://github.com/mlrun/mlrun/blob/master/README.md#installation).
+- Ensure that your cluster has a shared file or object storage for storing the data (artifacts).
 
 <a id="demo-sklearn-pipe"></a>
-## [End-to-End data prep + scikit-learn Pipeline with AutoML (Iris Data Set)](./sklearn-pipe/sklearn-project.ipynb)
```

Comment:
@Sharon-iguazio 1. title too long, 2. we lost the link to the notebook

Reply:
Regarding the NB link (for all demo doc sections) — we didn't lose the links; I replaced linking the section heading to a notebook — which is very non-standard and I would prefer to avoid this — with a relevant link right at the beginning of the demo description in each section. Al;so, I preferred to link to a dedicated demo README than to a notebook (the dedicated READMEs typically have more detailed info, or will have it in the future, and they have the necessary notebook links + this is also easier to maintain should there be changes to the demo notebooks in the future, and it's especially more handy in the case of demos with multiple notebooks).